test/spec/ol/interaction/modify.test.js

```diff
@@ -1000,7 +1084,7 @@ describe('ol.interaction.Modify', function () {
       simulateEvent('pointerdrag', 30, -5, null, 0);
       simulateEvent('pointerup', 30, -5, null, 0);
 
-      expect(circleFeature.getGeometry().getRadius()).to.equal(25);
+      expect(circleFeature.getGeometry().getRadius()).to.roughlyEqual(25, 1e9);
```

Comment:
The tolerance is way too high here, either should be 1e-9 or 0.1 like the others.

Reply:
Right, thanks for catching this! Fixed.